[2025-04-06T18:05:15.782+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_prediction_dag.predict_fraud manual__2025-04-06T18:05:12.230024+00:00 [queued]>
[2025-04-06T18:05:15.787+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_prediction_dag.predict_fraud manual__2025-04-06T18:05:12.230024+00:00 [queued]>
[2025-04-06T18:05:15.787+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-06T18:05:15.802+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): predict_fraud> on 2025-04-06 18:05:12.230024+00:00
[2025-04-06T18:05:15.805+0000] {standard_task_runner.py:60} INFO - Started process 138 to run task
[2025-04-06T18:05:15.808+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'fraud_prediction_dag', 'predict_fraud', 'manual__2025-04-06T18:05:12.230024+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/fraud_pipeline_dag.py', '--cfg-path', '/tmp/tmprp0x8ilq']
[2025-04-06T18:05:15.810+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask predict_fraud
[2025-04-06T18:05:15.840+0000] {task_command.py:423} INFO - Running <TaskInstance: fraud_prediction_dag.predict_fraud manual__2025-04-06T18:05:12.230024+00:00 [running]> on host edc87b335a79
[2025-04-06T18:05:15.891+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fraud_prediction_dag' AIRFLOW_CTX_TASK_ID='predict_fraud' AIRFLOW_CTX_EXECUTION_DATE='2025-04-06T18:05:12.230024+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-06T18:05:12.230024+00:00'
[2025-04-06T18:05:22.213+0000] {logging_mixin.py:188} INFO - STDOUT:
[2025-04-06T18:05:22.214+0000] {logging_mixin.py:188} INFO -  Нет новых транзакций
[2025-04-06T18:05:22.214+0000] {logging_mixin.py:188} WARNING - STDERR:
[2025-04-06T18:05:22.215+0000] {logging_mixin.py:188} WARNING -  
[2025-04-06T18:05:22.215+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-06T18:05:22.220+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fraud_prediction_dag, task_id=predict_fraud, execution_date=20250406T180512, start_date=20250406T180515, end_date=20250406T180522
[2025-04-06T18:05:22.242+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-06T18:05:22.250+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
